LeaderboardBackend.Test/Leaderboards.cs

```diff
@@ -339,6 +341,71 @@ public async Task DeletedBoardsDontConsumeSlugs()
         created!.CreatedAt.Should().Be(_clock.GetCurrentInstant());
     }
 
+    [Test]
+    public async Task RestoreLeaderboard_OK()
+    {
+        ApplicationContext context = _factory.Services.CreateScope().ServiceProvider.GetRequiredService<ApplicationContext>();
+
+        Leaderboard deletedBoard = new()
+        {
+            Name = "Super Mario World",
+            Slug = "super-mario-world-deleted",
```

Comment:
you'll need to set a unique slug here... 
The tests aren't transactional, so data is persisting - i think theres already been some discussion on how to fix this. I've looked, but its quite complicated in ASP.net to set this up...

Reply:
Yeah I realised they weren't at the time already; just forgot about it pretty much. I've simply swapped slugs since (wow the alliteration). And yeah IMO it's plain not worth the hassle to set that up for now.